app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -133,4 +155,13 @@ def build_connect_params(options)
 
     connect_params
   end
+
+  def host_handle(vm, options)
```

Comment:
@roliveri the second paragraph above is a little nebulous.  The auth info is defined in the host where? I thought maybe that "options" was known to the host object but unless I pass it in like
host_handle = vm.host.host_handle(options) it doesn't know the variable.